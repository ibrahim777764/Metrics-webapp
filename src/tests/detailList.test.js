import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store/configureStore';
import DetailsList from '../components/detailList';

describe('Country Component test', () => {
  it('Renders Deatils', () => {
    const app = renderer.create(
      <Provider store={store}>
        <DetailsList />
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
