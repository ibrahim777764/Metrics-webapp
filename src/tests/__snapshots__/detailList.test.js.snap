// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Country Component test Renders Deatils 1`] = `
<div>
  <div
    className="cou"
  >
    <div
      className="details"
    >
      <div
        className="contry-div"
      >
        <h1>
          Country:
        </h1>
      </div>
      <div
        className="info"
      >
        <div
          className="details-div"
        >
          <div
            className="dates"
          >
            <h2
              className="date-head"
            >
              DETAILS
            </h2>
            <p>
              <b>
                Date:
              </b>
              <b />
            </p>
          </div>
          <div
            className="main-wrap"
          >
            <div
              className="sor"
            >
              <p
                className="dat-type"
              >
                SOURCE:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p
                className="dat-type"
              >
                TOTAL CONFIRMED:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p
                className="dat-type"
              >
                TOTAL DEATH:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p
                className="dat-type"
              >
                CONFIRMED:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                TODAY DEATH:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                today_new_open_cases:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                today_new_recovered:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                today_open_cases:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                today_recovered:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                today_vs_yesterday_confirmed:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                today_vs_yesterday_deaths:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                today_vs_yesterday_open_cases:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                today_vs_yesterday_recovered:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                yesterday_confirmed:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                yesterday_deaths:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                yesterday_open_cases:
              </p>
              <b
                className="dat-type"
              />
            </div>
            <div
              className="sor"
            >
              <p>
                yesterday_recovered:
              </p>
              <b
                className="dat-type"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
